Orangenspannung/oestersjoen
# O03 create african borders

# What has changed:
* Created borders for Africa and the middle east
* Give OEN more ENG land
# How to test:
* Load up the game and check out Africa, you should not find any remnants of FRA and ENG on it anymore (also ITA)
* Do the same for the middle east
